docs/index.md

```diff
@@ -65,76 +69,76 @@ POST {json file} <url>/insert/
+    {"visit_id": 1234567891011121, "date": "2017-07-25 15:02:54.171-00:00", "location": "333 S State St", "provider": "John Doe"}, 
+    {"visit_id": 1234567891011121, "date": "2017-07-25 15:02:54.171-00:00", "location": "333 S State St", "provider": "John Doe"}, 
+    {"visit_id": 1234567891011121, "date": "2017-07-25 15:02:54.171-00:00", "location": "333 S State St", "provider": "John Doe"}, 
+    {"visit_id": 1234567891011121, "date": "2017-07-25 15:02:54.171-00:00", "location": "333 S State St", "provider": "John Doe"}, 
+    {"visit_id": 1234567891011121, "date": "2017-07-25 15:02:54.171-00:00", "location": "333 S State St", "provider": "John Doe"}, 
+    {"visit_id": 1234567891011121, "date": "2017-07-25 15:02:54.171-00:00", "location": "333 S State St", "provider": "John Doe"}
   ],
   "lab": [
-    {"lab_id": 1234567891011121, "route": "V", "healthcenterid": 5.0, "result": "None Detected ug/dL", "date": "2016-07-29", "type": "BLL"}, 
-    {"lab_id": 1234567891011121, "route": "V", "healthcenterid": 5.0, "result": "1", "date": "2015-08-21", "type": "BLL"}, 
-    {"lab_id": 1234567891011121, "route": "V", "healthcenterid": 5.0, "result": "1", "date": "2015-02-27", "type": "BLL"}, 
-    {"lab_id": 1234567891011121, "route": "V", "healthcenterid": 5.0, "result": "1", "date": "2014-06-06", "type": "BLL"}, 
-    {"lab_id": 1234567891011121, "route": "V", "healthcenterid": 1.0, "result": "3", "date": "2014-09-29", "type": "BLL"}, 
-    {"lab_id": 1234567891011121, "route": "V", "healthcenterid": 1.0, "result": "4.1", "date": "2014-01-09", "type": "BLL"}, 
-    {"lab_id": 1234567891011121, "route": "V", "healthcenterid": 1.0, "result": "4.1", "date": "2013-06-28", "type": "BLL"}, 
-    {"lab_id": 1234567891011121, "route": "V", "healthcenterid": 1.0, "result": "6.3", "date": "2013-03-13", "type": "BLL"}, 
-    {"lab_id": 1234567891011121, "route": "V", "healthcenterid": 1.0, "result": "9.7", "date": "2012-10-02", "type": "BLL"}, 
-    {"lab_id": 1234567891011121, "route": "V", "healthcenterid": 1.0, "result": "16.3", "date": "2012-06-19", "type": "BLL"}, 
-    {"lab_id": 1234567891011121, "route": "V", "healthcenterid": 1.0, "result": "3", "date": "2016-09-16", "type": "BLL"}, 
-    {"lab_id": 1234567891011121, "route": "V", "healthcenterid": 1.0, "result": "3", "date": "2015-12-15", "type": "BLL"}, 
-    {"lab_id": 1234567891011121, "route": "V", "healthcenterid": 1.0, "result": "3", "date": "2015-06-09", "type": "BLL"}, 
-    {"lab_id": 1234567891011121, "route": "V", "healthcenterid": 1.0, "result": "13.6", "date": "2012-03-20", "type": "BLL"}, 
-    {"lab_id": 1234567891011121, "route": "V", "healthcenterid": 1.0, "result": "15", "date": "2012-02-21", "type": "BLL"}, 
-    {"lab_id": 1234567891011121, "route": "V", "healthcenterid": 1.0, "result": "5", "date": "2011-08-19", "type": "BLL"}] 
+    {"lab_id": 1234567891011121, "type": "BLL", "date": "2016-07-29", "sample_type": "V", "result": "None Detected ug/dL"}, 
```

Comment:
Suggest a shorter lab array for the same reason as visit above.

Reply:
Agreed -- we've shortened to 3.